### PR TITLE
Release v0.8

### PR DESCRIPTION
**Pull request by dbrakenhoff**
Getting ready for a new nlmod release v0.8...

I'm a bit confused why commits from last year are showing up in the list...? Am I doing something wrong?

EDIT: it's just the first one I guess (https://github.com/gwmod/nlmod/pull/360/commits/cef2b4bc0e43bbc532cdc7f25afc56d2f8b7bf00), still confusing but less worrisome.

### CODE REVIEW

**Review by rubencalje — APPROVED** [review]

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -9,22 +9,24 @@ on:
...
-
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
+
     - name: Set up Python
-      uses: actions/setup-python@v4
+      uses: actions/setup-python@v5
       with:
         python-version: '3.9'
```
Comment (OnnoEbbens) [inline .github/workflows/python-publish.yml:19]:
```suggestion
        python-version: '3.11'
```

File: nlmod/plot/__init__.py [hunk 2]
```diff
@@ -7,6 +8,7 @@
     geotop_lithok_in_cross_section,
     geotop_lithok_on_map,
     map_array,
+    # modelextent,
```
Comment (OnnoEbbens) [inline nlmod/plot/__init__.py:11]:
```suggestion
```

**Review by OnnoEbbens — APPROVED** [review]
I only have 2 small suggestions

**Comment by bdestombe** [comment]
Great work guys! Sorry for not being able to put more time in it at the moment.

**Pull request merged**